templates/patterns/gallery/gallery.ui_patterns.yml

```
@@ -146,3 +146,27 @@ gallery:
       description: "Number of visible items in an expandable gallery."
       preview: 3
       escape: false
+    grid:
+      type: "boolean"
+      label: "Grid display"
+      description: "Display gallery as a grid. Default is false."
+      preview: true
+      escape: false
+    columns:
+      type: "numeric"
+      label: "Columns number"
+      description: "Number of columns. Default is 2. Only used if the grid display is enabled."
+      preview: 2
+      escape: false
+    ratio:
+      type: "string"
+      label: "Column number"
```

It's not the column number.
```suggestion
      label: "Aspect ratio"
```